javascript/utils/utils.js

```diff
@@ -55,6 +55,14 @@ export let canStatUpgrade = (planet, stat) => {
   return canUpgrade[stat];
 }
 
+const canCapture = (fromPlanet, toPlanet, maxEnergyPercent) => {
+  const energyNeededUponArrivalDoubled = planetToTake.energy * 2;
```

Comment:
where is planetToTake defined? 
also why not just energyCap? or a % of it

Reply:
Untested, but these are some ways I would probably go forward



  // easiest would be barest minimum to take it assuming its full and just exceed that by 1
const energyNeededUponArrivalDoubled = 1 + (toPlanet.energyCap * (toPlanet.defense / 100));
    
// or we could put ourselves on the curve would be barest minimum to take it assuming its full
const energyNeededUponArrivalDoubled = (toPlanet.energyCap * 0.15) + (toPlanet.energyCap * (toPlanet.defense / 100));

// or we need to calculate time it will take to get there, and use energyattime
let buffer = 90 * 1000; //90 seconds, takes forever for snarks to go through lets give ourselves time
let time = df.getTimeForMove(fromPlanet.locationId, toPlanet.locationId);
let energy = getEnergyAtTime(Date.now() + time + buffer)
const energyNeededUponArrivalDoubled = 1 + (energy * (toPlanet.defense / 100));